run.sh

```diff
@@ -15,6 +15,11 @@ function usage()
     echo "   start_zk    start the local single zookeeper server"
     echo "   stop_zk     stop the local single zookeeper server"
     echo "   format      check the code format"
+    echo "   publish     publish the program"
+    echo "   deploy      deploy the program to remote machine"
```

Comment:
To my understanding (correct me if I'm wrong), a complete remote deployment process should be:
`build and install`: common meaning, in this context it specially refers to `local env`
`publish`:  gather all necessary libs, exes, config files and scripts in one folder to be `ready` to send to remote machine, this operation generates one folder for each type of apps like meta, client and replica.
`deploy`: Send the specific folder to the corresponding target machine
`start`: Start the app on the remote machine